test/integration/sagemaker/util.py

```diff
@@ -60,11 +61,16 @@ def _production_variants(model_name, instance_type, accelerator_type):
 
     return production_variants
 
+
+def _test_bucket(region, boto_session):
+    account = boto_session.client('sts').get_caller_identity()['Account']
+    return f'sagemaker-{region}-{account}'
```

Comment:
what if the bucket doesn't exist?

Reply:
find_or_put_model_data will create the bucket